src/components/DeleteMovieDialog/DeleteMovieDialog.scss

```
@@ -7,7 +7,7 @@
   &__dialog {
     position: fixed;
     width: 100%;
-    height: 1787px;
+    height: 2000px;
```

And what if page content gets longer? Please familiarize yourself with [other units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units)

PS. I've noticed you have different modals, and for each, you are duplicating the styles. Ideally, it would be if you had one component called `Modal` that would consume the body and have common styles, but this is not the scope of this task. Hoverer in this ticket you could make things simpler by creating SCSS mixin and including it in each modal instead.

```suggestion
    height: 100vh;
```

---

I created mixins for modals and edited both components - MovieDialog and DeleteMovieDialog to unify styles along with BEM methodology implementation.